notebooks/sagemaker/rerank_v3_notebooks/Deploy rerank multilingual v3.0 model.ipynb

```diff
@@ -138,7 +138,7 @@
    "outputs": [],
    "source": [
     "co = Client(region_name=region)\n",
-    "co.create_endpoint(arn=model_package_arn, endpoint_name=\"cohere-rerank-multilingual-v3.0\", instance_type=\"ml.g4dn.xlarge\", n_instances=1)\n",
+    "co.create_endpoint(arn=model_package_arn, endpoint_name=\"cohere-rerank-multilingual-v3-0\", instance_type=\"ml.g5.2xlarge\", n_instances=1)\n",
```

Comment:
can you update the english notebook to match, please?
and let's use ml.g5.xlarge since that's 'vendor recommended' https://aws.amazon.com/marketplace/pp/prodview-ydysc72qticsw